pkg/alertmanager/operator.go

```diff
@@ -1040,6 +1044,21 @@ func (c *Operator) selectAlertmanagerConfigs(ctx context.Context, am *monitoring
 	return res, nil
 }
 
+func checkAlertmanagerRootConfig(ctx context.Context, conf *alertmanagerConfig) error {
+	if conf == nil || conf.Route == nil {
+		return nil
+	}
```

Comment:
Alertmanager wants at least one route, and that route needs a receiver.

Reply:
ok, I was trying to avoid a panic but armed with that knowledge I will improve the check :)